server/app/controllers/feed-object-controller.js

```diff
@@ -5,22 +5,48 @@ const { createUserError } = require('../../utils');
 const Joi = require('joi');
 
 /**
- * Fetch all the categories from the database.
+ * Fetch feedObjects from the database with pagination.
  *
  * @param req Request
  * @param res Response
  * @return {Promise.<void>} Nothing
  */
 async function getAll(req, res) {
-  if (req.query.offset && req.query.offset % 40 !== 0) {
-    throw createUserError('OffsetError', 'Offset must be a multiple of 40');
+  let offset = 0;
+  let limit = 40;
+
+  if (req.query) {
+    const schema = Joi.object().keys({
+      offset: Joi.number().integer().min(1),
+      limit: Joi.number().integer().min(1).max(100),
+    });
+
+    const { error } = schema.validate(req.query);
+    if (error) throw createUserError('BadRequest', error.message);
+
+    // Cast to number
+    offset = +req.query.offset || 0;
+    limit = +req.query.limit || 40;
   }
```

Comment:
Here, `req.query` will always be true (I think an empty object is set). You could just let Joi validate the object, and after checking, you set your default values

Reply:
Ok I don't understand